lms/djangoapps/learner_dashboard/views.py

```diff
@@ -46,7 +46,7 @@ def program_listing(request):
 def program_details(request, program_uuid):
     """View details about a specific program."""
     programs_config = ProgramsApiConfig.current()
-    if not programs_config.show_program_details:
+    if not programs_config.enabled:
```

Comment:
I actually disagree.  I think we're eliminating the concept of having discrete enabling of units of programs and switching to a single programs? (y/n) model.  I think the code looking at that field directly is logically consistent and _more_ clear.